repose-aggregator/components/filters/valkyrie-authorization-filter/src/test/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilterTest.scala

```diff
@@ -596,26 +618,32 @@ class ValkyrieAuthorizationFilterTest extends FunSpec with BeforeAndAfterEach wi
     val transformedTenant = "98765"
     val contactId = "123456"
     val filterChain = mock[FilterChain]
-    val mockServletRequest = new MockHttpServletRequest
-    val mockServletResponse = new MockHttpServletResponse
+    var mockServletRequest = new MockHttpServletRequest
```

Comment:
in the diff it looked like they were now at the class level with the same name, so the local one's seemed pointless. Could have been a bad diff though.